### PR TITLE
feat: connector client: switch PolicyService

### LINKED ISSUE #58: connector client: switch `PolicyService`
# Feature Request

Use the EDC-connector-client in the `PolicyService`

## Which Areas Would Be Affected?

_e.g., DPF, CI, build, transfer, etc._

## Why Is the Feature Desired?

_Are there any requirements?_

## Solution Proposal

_If possible, provide a (brief!) solution proposal._

### PR DESCRIPTION
**Pull request by janpmeyer**
## What this PR changes/adds

Use EDC-connector-client for `ContractDefinition

## Why it does that

_Briefly state why the change was necessary._

## Further notes

_List other areas of code that have changed but are not necessarily linked to the main feature. This could be method
signature changes, package declarations, bugs that were encountered and were fixed inline, etc._

## Linked Issue(s)

Closes #58 and Closese #65

### CODE REVIEW

**Comment by janpmeyer** [comment]
At the current state on creating a new Policy the values are not used given in the Dialog.

Also i have the question about the assignee and assigner of a policy. Does the still exist? Because i don't see this fields in the management-Api Documentation. Or i just did not see it? @ndr-brt

**Comment by ndr-brt** [comment]
@janpmeyer they are part of the ODRL model, but they are not used by the EDC, so they can be avoided

**Comment by janpmeyer** [comment]
@ndr-brt fixed now the id of an policyDefiniton and also the permission field of the policy. But the fields prohibition and obligation are not working yet

**Comment by janpmeyer** [comment]
With the new EDC-Connector-Client Beta-6 obligations and prohibitions are also working.

File: src/modules/edc-demo/components/contract-definition-editor-dialog/contract-definition-editor-dialog.component.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
 import {
   AssetService, PolicyDefinitionResponseDto, PolicyService
```
Comment (ndr-brt) [inline src/modules/edc-demo/components/contract-definition-editor-dialog/contract-definition-editor-dialog.component.ts:4]: the `PolicyDefinitionResponseDto` can be removed now

**Review by ndr-brt — APPROVED** [review]

**Pull request merged**